pandas/core/frame.py

```diff
@@ -6837,6 +6837,32 @@ def reorder_levels(self, order: Sequence[Axis], axis: Axis = 0) -> DataFrame:
         Returns
         -------
         DataFrame
+        
+        Examples
+        --------
+        >>> data={'Name':['Rahul', 'Arijit', 'Tushar'],
+        ...      'Mark':[70,95,89],
+        ...      'Branch':['CSE', 'EEE', 'IT'],
+        ...      'Grade':['C', 'B', 'A'], 
+        ... }
+        >>> df=pd.DataFrame(data, columns = ['Mark', 'Branch', 'Grade', 'Name'])
+        >>> df.set_index(['Branch', 'Mark', 'Grade'], inplace = True)
```

Comment:
also, can you please write this as `df = df.set_index(['Branch', 'Mark', 'Grade'])`?

Reply:
Ok I will do the changes.